collections/nemo_nlp/nemo_nlp/data/datasets/token_classification.py

```diff
@@ -218,11 +222,16 @@ def __init__(self,
             data_dir = os.path.dirname(text_file)
             filename = os.path.basename(text_file)[:-4]
```

Comment:
comment to explain that you expect file to end with '.txt' and that's why [:-4]

Reply:
done